packages/typescript/src/plugins/FederatedTypesStatsPlugin.ts

```diff
@@ -21,19 +22,24 @@ export class FederatedTypesStatsPlugin {
         async () => {
           const { typesIndexJsonFileName, publicPath } = this.options;
 
+          const typesIndexJsonFilePath = path.join(
```

Comment:
Move this logic to `normailzeOptions`.

Reply:
Thanks for the review! Updated. This seems like the right fix for Next right? We'd expect the chunks directory to host all of the files generated by the typescript plugin - in the case of next?

Reply:
Yes, this seems like a right fix.